### ISSUE TITLE
Poor Contrast Ratio (red on white)

### ISSUE THREAD

**Issue by pbailie** [rpi-dotcio/phpLicenseWatcher#22]
Red `#ff0000` on white `#ffffff` has a contrast ratio of 3.99:1.  We should try to achieve a contrast ratio of at least 4.5:1.  As this project develops, any instance of red on white should be changed to use a darker red.

I propose crimson `#dc143c` as it is dark enough to achieve contrast ratio of 4.99:1.

Despite this CSS color being defined by W3C, Chrome is interpreting crimson as `#ed143d` which has contrast ratio 4.4:1.  Therefore, crimson should be coded by value rather than by name.  

```
color: #dc143c;
color: rgb(220, 20, 60);
```

If we need to use a darker red, firebrick `#b22222` has a contrast ratio of 6.67:1.
